apps/modernization-ui/src/apps/patient/add/extended/nav/extended-patient-nav.module.scss

```diff
@@ -4,6 +4,7 @@
 aside {
     position: sticky;
     top: 0;
+    width: 30%;
```

Comment:
Issue: Figma has the nav as `12.1875rem`. Not 30%

![image](https://github.com/user-attachments/assets/589fb416-545a-4734-83f1-9b029c7b05cd)